src/commands/pause.js

```diff
@@ -16,0 +1,15 @@
+const Player = require('../audio/player.js');
+
+module.exports = {
+    name: 'pause',
+    run: function(event) {
+        let player;
+        if (!Player.getSendingPlayer(event.guild)) {
+            return event.channel.send('❌ 必須要有音樂正在播放');
+        } else {
+            player = Player.getSendingPlayer(event.guild);
+            if (!event.allowModify) return event.channel.send('❌ 你必須加入一個語音頻道');
+        }

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779465)
```suggestion
        }
```